cllc-public-app/ViewModels/License.cs

```diff
@@ -36,5 +36,16 @@ public class License
         public DateTimeOffset? expiryDate { get; set; }
 
         public List<string> endorsements { get; set; }
+
+        public string representativeFullName { get; set; }
```

Comment:
These should all begin with a capital letter.  Note that the way dotnet mvc works is when the json encoder processes a model it will convert that first letter to lower case.  

I know that License is not following this convention, but it should.  This might be a good time to align it.

Reply:
great I will update all to capital. I was trying to align with the current License but would rather Capital